src/routing/pages/probe/Table.svelte

```diff
@@ -4,14 +4,14 @@
   import Probe from '../../wrappers/Probe.svelte';
   import ProbeTableView from '../../../components/table/ProbeTableView.svelte';
   import ProbeTitle from '../../../components/regions/ProbeTitle.svelte';
-  import { store, probe } from '../../../state/store';
+  import { store } from '../../../state/store';
 </script>
 
 <Probe let:data let:probeType>
   <div class="graphic-body__content graphic-body__content--no-padding">
     <div in:fade>
       <!-- this conditional is a stopgap until we fix https://github.com/mozilla/glam/issues/206 -->
-      {#if $probe.type}
+      {#if $store.probe.loaded}
```

Comment:
It looks like the issue mentioned in the comment has been resolved. Is this conditional still needed? If so, maybe we can update the comment explaining why.